packages/node/src/lib/feeds/feeds.spec.ts

```diff
@@ -32,7 +32,7 @@ describe('Use of Novu node package - Feeds class', () => {
 
     expect(mockedAxios.post).toHaveBeenCalled();
     expect(mockedAxios.post).toHaveBeenCalledWith('/feeds', {
-      name: 'test',
+      name: 'test-feeds',
```

Comment:
This test was failing.
I guess we are not running the package integration tests in the CI. 😭